src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    @FindBy (css = "[name='name']")
+        WebElement inputNameField;
+    @FindBy (css = "div.success.show")
+        WebElement successMessage;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```

![POM_vs_Factory_1](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/002791a7-5645-49ec-bd07-03b2a1707562)

![POM_vs_Factory_2](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d76b4c96-b845-4aac-bdf9-ff89a6a0160e)